apps/api/src/app.ts

```diff
@@ -1,26 +1,29 @@
-import cors from "cors";
-import express from "express";
+import cors from 'cors';
+import express from 'express';
+import admin from 'firebase-admin';
 
-import { CharactersController } from "./controllers/characters-controller";
-import { GameController } from "./controllers/game-controller";
-import { GamesController } from "./controllers/games-controller";
-import { ScoresController } from "./controllers/scores-controller";
-import { UserController } from "./controllers/user-controller";
-import { UsersController } from "./controllers/users-controller";
+import { CharactersController } from './controllers/characters-controller';
+import { GameController } from './controllers/game-controller';
+import { GamesController } from './controllers/games-controller';
+import { ScoresController } from './controllers/scores-controller';
+import { UserController } from './controllers/user-controller';
+import { UsersController } from './controllers/users-controller';
 
 const app = express();
 
+admin.initializeApp();
```

Comment:
I had to do this change because in the `on-new-user-created.trigger.ts` I am now using the HttpsError (to be able to do something with the error message in the web application) which threw the following error: 
> The default Firebase app does not exist. Make sure you call initializeApp() before using any of the Firebase services. at FirebaseAppError

After some digging this was a solution that seemed to work, although I am not sure about why it's needed when using the HttpsError 🤔